docs/index.rst

```diff
@@ -9,55 +9,100 @@ If you want to build a completely different site with all the same Django
 optimizations for security, scalability, L10n, and ease of use, check out
 Mozilla's `Playdoh starter kit <http://playdoh.readthedocs.org/>`_.
 
-
 .. _install-with-docker:
 
 Quickstart
 ----------
+Want the easiest way to start contributing to AMO? Try our docker-based
+development environment.
```

Comment:
I would like suggest we make it the only supported way (other than for what ops do for prod). Having everything run just one way makes it much, much, much easier.